esmvalcore/preprocessor/_multimodel.py

```diff
@@ -302,6 +302,12 @@ def _combine(cubes):
 
     cubes = CubeList(cubes)
 
+    # For a single cube, merging returns a scalar CONCAT_DIM, which leads to a
+    # "Cannot collapse a dimension which does not describe any data" error when
+    # collapsing. Thus, treat single cubes differently here.
+    if len(cubes) == 1:
```

Comment:
we need a check that looks up the way this function is called ie if `ensemble_statistic` or any other case where single-cube stats makes sense, then it's OK, otherwise we need it to barf, since single-cube stats when multi-cube stats should happen is a bad thing, and a statistician would stab you :hocho:

Reply:
scratch that :arrow_up: for now